test/__snapshots__/modules-option.test.js.snap

```diff
@@ -5362,7 +5362,7 @@ Array [
   color: blue;
 }
 
-._1wABXM_RabWHj--wsPrhvM {
+.green {
```

Comment:
Need investigate why it is happens

Reply:
its supposed to happen. this is an effect of the change, `other` is an imported value

Reply:
@jquense let's rename selector, because it can be misleading (just add couple character)

Reply:
i can, but this test is specifically testing the css-modules resolution logic. it really should include a resolution in a selector, I would consider the test incomplete otherwise, but if you want it to be a different test i can do that

Reply:
@jquense, i agree with you, just want to avoid misleading for other developers which see PR and tests